ivy/functional/frontends/numpy/mathematical_functions/trigonometric_functions.py

```diff
@@ -44,3 +44,46 @@ def tan(
 
 
 tan.unsupported_dtypes = {"torch": ("float16",)}
+
+
+def arcsin(
+    x,
+    /,
+    out=None,
+    *, 
+    where=True, 
+    casting='same_kind', 
+    order='K', 
+    dtype=None, 
+    subok=True
+):
+    if dtype:
+        x = ivy.astype(ivy.array(x), ivy.as_ivy_dtype(dtype))
+    ret = ivy.arcsin(x, out=out)
+    if ivy.is_array(where):
+        ret = ivy.where(where, ret, ivy.default(out, ivy.zeros_like(ret)), out=out)
+    return ret
+
+
+arcsin.unsupported_dtypes = {"torch": ("float16",)}
+
+def arccos(
+    x,
+    /,
+    out=None, 
+    *, 
+    where=True, 
+    casting='same_kind', 
+    order='K', 
+    dtype=None, 
+    subok=True
+):
+    if dtype:
+        x = ivy.astype(ivy.array(x), ivy.as_ivy_dtype(dtype))
+    ret = ivy.arccos(x, out=out)
```

Comment:
This should be changed to `ivy.acos` instead of `ivy.arccos` since the function exists under the name `acos` in the ivy namespace.